{
  "Header": {
    "Original": "CVE-2021-20206 (An improper limitation of path name flaw was found in containernetwork ...)",
    "Line": 149576,
    "ID": "CVE-2021-20206",
    "Description": "(An improper limitation of path name flaw was found in containernetwork ...)"
  },
  "Annotations": [
    {
      "Original": "- golang-github-appc-cni 0.8.1-1 (bug #983659)",
      "Line": 149577,
      "Type": "package",
      "Package": "golang-github-appc-cni",
      "Kind": "fixed",
      "Version": "0.8.1-1",
      "BugNo": 983659
    },
    {
      "Original": "[buster] - golang-github-appc-cni \u003cpostponed\u003e (Limited support, minor issue)",
      "Line": 149578,
      "Type": "package",
      "Release": "buster",
      "Package": "golang-github-appc-cni",
      "Kind": "postponed",
      "Description": "Limited support, minor issue"
    },
    {
      "Original": "[stretch] - golang-github-appc-cni \u003cno-dsa\u003e (Minor issue)",
      "Line": 149579,
      "Type": "package",
      "Release": "stretch",
      "Package": "golang-github-appc-cni",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/containernetworking/cni/pull/808",
      "Line": 149580,
      "Type": "NOTE",
      "Description": "https://github.com/containernetworking/cni/pull/808"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1919391",
      "Line": 149581,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1919391"
    }
  ]
}